clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy

```diff
@@ -220,6 +219,16 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
       .build()
   }
 
+  long getCreationTime(Stack stack) {
```

Comment:
My two cents is that I would create a parseZonedDateTime(String time, ZonedDateTime defaultTime) inside of DateUtils in case other operations or places need this.